Datadog/TargetSupport/Example/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CustomLogsURL</key>
+	<string>https://$(CUSTOM_LOGS_URL)</string>
+	<key>CustomRUMURL</key>
+	<string>https://$(CUSTOM_RUM_URL)</string>
```

Comment:
when you put a value like `https://foo.bar` in `xcconfig`, `foo.bar` is taken as comments
when you try `https:\/\/foo.bar` then it's passed as `http:\/\/foo.bar` instead of `https://foo.bar`

as a workaround, now i take `foo.bar` from `xcconfig` and add `https://` as prefix in the code (`Environment.swift`)